Firmware/planner.cpp

```diff
@@ -1054,9 +1054,9 @@ Having the real displacement of the head, we can calculate the total movement le
         if (e_D_ratio > 3.0)
             block->use_advance_lead = false;
         else if (e_D_ratio > 0) {
-            const float max_accel_per_s2 = cs.max_jerk[E_AXIS] / (extruder_advance_K * e_D_ratio);
-            if (cs.acceleration > max_accel_per_s2) {
```

Comment:
Looks like there was a bug here previously? When I compare the code to Marlin 2. Here we didn't allow  `cs.travel_acceleration` or `cs.retract_acceleration`. While Marlin 2 doesn't do this, it uses `accel` as I propose in the PR.

Reply:
In other words: `block->acceleration_st` could be equal to `cs.travel_acceleration` or `cs.retract_acceleration`. But if the check `cs.acceleration > max_accel_per_s2` was false (print acceleration), then we didn't apply reduce/limit the acceleration correctly in cases where `cs.travel_acceleration > cs.acceleration` or `cs.retract_acceleration > cs.acceleration`

Reply:
I'd be cautios in case of Linear Advance. @wavexx fixed a lot of bugs in our code compared to Marlin2.

Reply:
It look correct to me. We added separate travel acceleration later, so this was definitely an oversight!